### PR TITLE
Add e2e test 

### LINKED ISSUE #455: Add integration test with pysqlflow
SQLFlow repo is responsible for releasing the final image. So its CI should contain integration tests with other repositories at `sql-machine-learning`. Recent bug https://github.com/sql-machine-learning/sqlflow/issues/453 exposes the lack of these type of tests.

At the time of writing, our tests include integration with gohive and models. We should add the integration test with pysqlflow.

**Proposed Solution**

1. Fix pysqlflow version in Dockerfile
1. Add test cases in `cmd/sqlflowserver/main_test.go`. e.g. `cmd.Run("python", "-c", "import sqlfow ...")`

### PR DESCRIPTION
**Pull request by Yancey0623**
Fixed #455 

Refer iPython unit test implementation: https://github.com/ipython/ipython/blob/master/IPython/core/tests/test_magic.py

### CODE REVIEW

File: scripts/test_e2e.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+service mysql start
+
+
+go generate ./...
+go get -v -t ./...
+go install ./...
+
+DATASOURCE="mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0"
+sqlflowserver --datasource=${DATASOURCE} &
```
Comment (typhoonzero) [inline scripts/test_e2e.sh:26]: Need to kill this `sqlflowserver` process when the test is finished, because mutiple test scripts need to run.

**Review by typhoonzero — APPROVED** [review]

File: scripts/test_e2e.sh (hunk 1 above)
Comment (tonyyang-svail) [reply scripts/test_e2e.sh]: Looks like `test_e2e.sh` is ran in an isolated container. I am wondering: after the execution of `ipython sql/python/test_magic.ipy`, will Docker stop the container by killing all the running process?

**Review by tonyyang-svail — APPROVED** [review]
LGTM++

File: scripts/test_e2e.sh (hunk 1 above)
Comment (tonyyang-svail) [reply scripts/test_e2e.sh:26]: Looks like `test_e2e.sh` is ran in an isolated container. I am wondering: after the execution of `ipython sql/python/test_magic.ipy`, will Docker stop the container by killing all the running process?

Comment (Yancey0623) [reply scripts/test_e2e.sh:26]: Yes, The Travis would run these test scripts in the separate Docker containers, maybe we can execute these test in one Docker container in the future.

**Pull request merged**